### ISSUE TITLE
plot.scatter and plot.quiver missing in Documentation

### ISSUE THREAD

**Issue by sikersten** [pyfar/pyfar#345]
## General

* pyfar version: 0.4.3

## Description

The plot functions ``scatter`` and ``quiver`` are missing in the documentation on https://pyfar.readthedocs.io/en/v0.4.3/modules/pyfar.plot.html#.
The init file in `plots` looks ok for me. Does anybody know the reason?

**Comment by f-brinkmann** [comment]
I think we wrote these plots especially for the Coordinates and Orientations Classes where they should also be documented

**Comment by sikersten** [comment]
Yes, you are right, they are used by` coordinates.show()` and `orientations.show()`. The possibility of calling them e.g. as `scatter(coordinates)` is not documented though.
Do we want to change that?

**Comment by f-brinkmann** [comment]
I would suggest to either make them private or re-structure to make them accept Coordinates and Orientations objects. The latter might be more consistent, because the other plot functions are not implemented as class functions. In this case we could consider removing the class functions with a deprecation warning.

**Comment by f-brinkmann** [comment]
We discussed to leave as is for now and probably move the functions to the plot module with an additional option to plot data as a color/size coded scatter plot.
